docs/openapi.yaml

```diff
@@ -1,15 +1,25 @@
+
 openapi: 3.0.2
+
+
 schemes:
-  - https
+  - "http"
+  - "https"
+
 produces:
-  - application/json
+  - "application/json"
+  - "text/plain"
+
 consumes:
-  - application/json
+  - "application/json"
+  - "text/plain"
+
+
 info:
   title: IUDX Data Ingestion Server APIs
   version: 3.5.0

```

Comment:
can version be changed to 4.0.0 here and all api servers? @kailash

Reply:
yes, will change this in all API docs